src/smart-switch.js

```diff
@@ -84,11 +85,15 @@ class SmartSwitch extends BasePlugin {
     }
   }
 
-  _doRequest(resource: string): Promise<any> {
+  async _doRequest(resource: string): Promise<any> {
     const responseTimeoutHandler = () => {
       this.logger.warn(`Timeout reached ${this.config.responseTimeoutSec} seconds, loading original source`);
     };
 
+    if (this.config.followRedirects) {
+      resource = await this._extractResource(resource);
```

Comment:
it should run recursively to get the final resources in case of multi redirections

Reply:
it done automatically by the browser

Reply:
per [mdn](https://developer.mozilla.org/en-US/docs/Web/API/fetch) it's not supported in ie11. I guess there is polyfill you can use

Reply:
polyfill added